src/data.js

```diff
@@ -1,6 +1,46 @@
 // keep it commonjs export
 // Data is validated against the following schema https://github.com/wesbos/awesome-uses/blob/master/scripts/utils.js#L53-L68
 module.exports = [
+  {
+    name: 'Slim Ktari',
+    description: "I code so I'm.",
+    url: 'http://slimktari.com/',
```

Comment:
The url "http://slimktari.com/" fails to match the required pattern: /(use|uses|using|setup|environment|